tests/test_mlpipeline.py

```diff
@@ -549,6 +549,113 @@ def test_get_output_variables(self):
 
         assert names == ['a_variable']
 
+    @patch('mlblocks.mlpipeline.MLBlock', new=get_mlblock_mock)
```

Comment:
Good testing! Would you mind also adding a unit test for the new `_get_block_variables` method?
I think that it's enough to add only one case, no need to add the three of them.